src/vmm/src/config.rs

```diff
@@ -48,23 +153,11 @@ impl TryFrom<String> for MemoryConfig {
 
     fn try_from(mem_cfg_str: String) -> result::Result<Self, Self::Error> {
         // Supported options: `size=<u32>`
-        let mem_cfg_str_lower = mem_cfg_str.to_lowercase();
-        let tokens: Vec<&str> = mem_cfg_str_lower
-            .split('=')
-            .filter(|tok| !tok.is_empty())
-            .collect();
-        if tokens.len() != 2 {
-            return Err(ConversionError::ParseMemory(mem_cfg_str));
-        }
-        if tokens[0] != "size_mib" {
-            return Err(ConversionError::ParseMemory(tokens[0].to_string()));
-        }
-        tokens[1]
-            .parse::<u32>()
-            .map(|mem_size_mib| MemoryConfig {
-                size_mib: mem_size_mib,
-            })
-            .map_err(|_| ConversionError::ParseMemory(tokens[1].to_string()))
+        let mut tokens = tokenize(mem_cfg_str);
```

Comment:
Related to `tokenize` comment above, how about calling it a - `config_map` or even more explicit like `mem_cfg_map`, `net_cfg_map` etc. ?

Reply:
I see your point on the map vs vec, but "config_map" is a bit misleading IMO. This map contains tokens that if correctly consumed will lead to a config_map, but at this point that map contains just an intermediate (possibly invalid) representation. Would `token_map` be suitable for you? Also, for reducing cognitive load and ease copy/pasting, I would call it in a "generic" way, thus "config_map" or "tokens_map" and so on.

Reply:
well - I don't think a generic name is needed, since this call is in the `try_from` for each of the configs - so something like `mecfg_params_map` or better still `memcfg_params` is good enough because the original function already tells it's a map.  I would've called it something like `memcfg_params = cfg_args_to_map(...)`  - clearly conveys. But it's just an opinion. I am fine, if you prefer `token_map`.